projects/storefrontapp-e2e-cypress/cypress.ci.json

```diff
@@ -11,7 +11,7 @@
   },
   "videoUploadOnPasses": false,
   "env": {
-    "API_URL": "https://20.83.184.244:9002",
+    "API_URL": "https://spartacus-fdev8.eastus.cloudapp.azure.com:9002",
```

Comment:
Will update `.env-cmdrc` and cypress.*.json accordingly before merging ONCE the CI server gets updated with latest sample data.